action.yml

```diff
@@ -0,0 +1,21 @@
+name: setup-nucleus-cli
+description: Setup Nucleus CLI, on GitHub Actions runners
+inputs:
+  version:
+    description: Version of Nucleus CLI to install
+    required: false
+    default: 0.0.26
```

Comment:
if we add the latest, we can just set this to `default: latest` - then we will almost never have to update this GH action.